Automattic/jetpack
# Update ThreatsDataView to use js-packages IconTooltip

## Description
Replace our custom `IconTooltip` component with the js-packages alternative

## Proposed changes:

* Adds a `hoverShow` prop and hover handling to the base component to
* Replaces the component in the `ThreatsDataView` component

### Other information:


## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1475

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch
* Load storybook with `cd projects/js-packages/storybook && npm run storybook:dev`
* View the Threats Data View component
* Ensure the content is positioned and styled appropriately and the popover displays when hovering on/off of the red info icon
* Review the code and ensure the implementation is correct

## Screen capture
![Screen Capture on 2024-10-22 at 08-59-50](https://github.com/user-attachments/assets/fca76370-2007-4ce8-af84-4d188db3abd6)
